app/models/form.rb

```diff
@@ -1,5 +1,6 @@
 class Form < ApplicationRecord
   has_many :pages, -> { order(position: :asc) }, dependent: :destroy
+  has_many :published_forms
```

Comment:
What's the benefit of calling this `published_forms` instead of `live_forms`?

Reply:
I think `live_versions` could be confusing because a dev may assume that form filler will be able to use any record for their submissions when we really only want one live version per form.  This table will have every single live form that could have ended up on GOV.UK. We dont know for sure because its a manual process but there is a possibility that they were "published" on GOV.UK at some point in time. Perhaps if we were only keeping one single record then I would agree with you that `live_version` would seem more appropriate.

Reply:
I'd prefer to reserve use of the word "publish" when we're referring to the activity relating to publishing on GOV.UK. We don't use the word "publish" elsewhere in the team, so following the concept of a ubiquitous language we'd not use it here to prevent possible confusion.

Reply:
I think this ties back to what @james-sheppard-gov-uk mentioned about reserving the term 'published' for things that have go through the publishing process for GOV.UK. 

It may be the case that somewhere down the line we would want to distinguish between forms that are live and forms that are currently published (maybe for metrics or something)

Reply:
Even though it's potentially clunky I'd prefer something like `live_forms` or `made_live_forms` here - we've been quite careful to draw the distinction between published and live elsewhere so I think it would be useful to preserve that distinction.

Reply:
I've now renamed `published_forms` table to `made_live_forms`